test/e2e/utils.go

```diff
@@ -0,0 +1,97 @@
+/*
+Copyright 2024.
```

Comment:
nit: 2025

```suggestion
Copyright 2025.
```

Reply:
You marked this comment resolved, but I still see 2024 instead of 2025. Probably you forgot to push your local change?